### PR TITLE
Add custom RPMs utility

### PR DESCRIPTION
**Pull request by sabinaaledort**
The custom RPMs script allows to install RPMs from an external source on a node.
One main use of this script is to install RT kernel on a node.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -63,6 +63,12 @@ The full test suite can be found at [https://github.com/openshift/openshift-test
 
 - optionally run `ORIGIN_TESTS_REPOSITORY=test.repository.com:5000/origin-tests make origin-tests-disconnected-environment` to mirror all the required test images to a container image repository and source required test images from your repository when running origin-tests in a disconnected environment.
 
+### Custom RPMs
+The custom RPMs utility allows to install RPMs from an external source on a node.
```
Comment (yuvalk) [inline README.md:67]: on a Openshift node ?
on a RHCOS/FCOS based node?

File: README.md [hunk 2]
```diff
@@ -63,6 +63,12 @@ The full test suite can be found at [https://github.com/openshift/openshift-test
...
+RPMS_SRC (RPMs download source URL) must be provided.
+REMOVE_PACKAGES should be set when installing RT kernel RPMs to override the installed kernel packages.
+RPMS_NODE_ROLE is optional and defaults to 'node-role.kubernetes.io/worker'.
```
Comment (yuvalk) [inline README.md:70]: missing here somewhere the instruction to set these and run `make custom-rpms` or something

File: hack/custom_rpms.sh [hunk 3]
```diff
@@ -0,0 +1,123 @@
...
+REMOVE_PACKAGES="${REMOVE_PACKAGES:-}"
+RPMS_NODE_ROLE="${RPMS_NODE_ROLE:-node-role.kubernetes.io/worker}"
+
+if [ "$RPMS_SRC" == "" ]; then
+	echo "[ERROR]: No RPMS_SRC provided"
+	exit 1
+fi
+
+cat <<EOF | oc apply -f -
+kind: ConfigMap
```
Comment (yuvalk) [inline hack/custom_rpms.sh:17]: can we have the ConfigMap and DaemonSet be their own files, maybe under features-config, like any other feature?

Comment (yuvalk) [reply hack/custom_rpms.sh:17]: the environment variables (RPMS_SRC and REMOVE_PACKAGES) can be env in the daemon set, then they can be easily overriden by kustomize.

hmm, that wont fly with your changes detection mechanism (cause you count on the configmap to change)
so maybe create an additional ConfigMap with just these two?

**Comment by yuvalk** [comment]
/lgtm
/approve
/hold
let's wait till we branch out 4.8

**Comment by sabinaaledort** [comment]
/test e2e-gcp-ovn
the failure is being investigated

**Comment by sabinaaledort** [comment]
/test e2e-gcp-ovn
it failed due to docker pull rate limit which will be fixed in:
https://github.com/openshift/release/pull/19776

**Comment by sabinaaledort** [comment]
/test e2e-gcp-ovn
it failed due to docker pull rate limit which will be fixed in:
openshift/release#19776

**Comment by sabinaaledort** [comment]
/test e2e-gcp-ovn
flaky ovs_qos tests

**Comment by sabinaaledort** [comment]
/test e2e-gcp-ovn
it failed due to docker pull rate limit which will be fixed in:
openshift/release#19776

**Comment by yuvalk** [comment]
/retest
infra issues

**Pull request merged**